dom.bs

```diff
@@ -744,11 +744,16 @@ steps:
 
  <li>
   <p>For each <var>tuple</var> in <a>context object</a>'s <a for=Event>path</a>:
-
-  <ol><li><p>If <var>currentTarget</var> is a <a>node</a> and <var>tuple</var>'s <b>item</b> is not
-  <a>closed-shadow-hidden</a> from <var>currentTarget</var>, or <var>currentTarget</var> is not a
-  <a>node</a>, then append <var>tuple</var>'s <b>item</b> to <var>composedPath</var>.</p></li></ol>
-
+  <ol>
+   <li><p>If <var>currentTarget</var> is a {{Window}}, then:
```

Comment:
Also needs a newline before the `<p>` since its not the only child of the `<li>`